drivers/dai/intel/ssp/ssp.h

```diff
@@ -7,6 +7,22 @@
 #ifndef __INTEL_DAI_DRIVER_SSP_H__
 #define __INTEL_DAI_DRIVER_SSP_H__
 
+#define SSP_IP_VER_1_0 0x10000 /* cAVS */
+#define SSP_IP_VER_1_5 0x10500 /* MTL */
+#define SSP_IP_VER_2_0 0x20000 /* LNL */
+#define SSP_IP_VER_3_0 0x30000 /* ACE30 */
+
+/* SSP IP version defined by CONFIG_SOC*/
+#if defined(CONFIG_SOC_SERIES_INTEL_ADSP_CAVS)
+#define SSP_IP_VER SSP_IP_VER_1_0
+#elif defined(CONFIG_SOC_INTEL_ACE15_MTPM)
+#define SSP_IP_VER SSP_IP_VER_1_5
+#elif defined(CONFIG_SOC_INTEL_ACE20_LNL)
+#define SSP_IP_VER SSP_IP_VER_2_0
+#elif defined(CONFIG_SOC_INTEL_ACE30)
+#define SSP_IP_VER SSP_IP_VER_3_0
+#endif
```

Comment:
just suggestion: 
```
#else 
#error ...unknown HW...
#endif 
```

Reply:
Yes, this is a good idea - I will change it. Thanks